### DISCUSSION TITLE
2023-11-01 Meeting Agenda, Recording, and Notes

### DISCUSSION THREAD

**Discussion by rfay** [ddev/ddev#5483] [DDEV Advisory Group]
Previous meeting: 
* #5319

Next meeting: 6 Mar 2024 (Likely a review-and-planning longer meeting)

## Agenda

* Turn on recording, ask for a minutes-taker

### All things DDEV
  * Stas Zhuk (@stasadev) is now a full maintainer, and paid by the DDEV Foundation, see [blog](https://ddev.com/blog/introducing-maintainer-stas). His goal is to work about 10 hours/week.
  * Prolific maintainer Simon Gilli (@gilbertsoft) is out of touch, hopefully temporarily.  I haven't been able to reach him, have even tried to get him by phone.
  * DDEV v1.22.3 and v1.22.4 released
  * [Live Contributor/Maintainer trainings](https://ddev.com/blog/contributor-training/) have continued.
  * Randy spoke at Craft CMS Dot All and did 5 BoFs and saw so many people at Drupalcon Lille
  * Randy will be gone December and January, ticket has been purchased. This should be a great opportunity for growth and maturing in the DDEV community. 
  * DDEV has been proposed and soft-approved as "official" local development environment for Drupal. Thanks @deviantintegral . DDEV is already the official local dev environment for TYPO3, Craft CMS, and Ibexa.
  * Marketing efforts have slowed, but current income is keeping pace with current needs. 
  * Docker Sponsored Open Source program has been renewed for (nearly obsolete) `drud` org, will be renewed for `ddev` org next year. 
  * Everybody here should be seeing the monthly newsletter come through, please let me know if you're not getting it or it's landing in spam.

### Key Goals for November
* **Maintainer training plan and docs**. (We already have a private repo with a number of videos and docs for obscure activities that aren't covered in the regular docs or might be sensitive.)

### Financial Matters: 
  * The bank is working on adding Mike Anello (@ultimike) as a signer on the DDEV Foundation bank account. This means that if Randy were not available in any way, Mike could do the right thing with the funds in the account. This is a small first step toward more formal governance with DDEV.
  * Financial status/reports: We have about $17,000 in the bank.
  * A number of sponsors have come forward, including Amazee.io sponsoring Lagoon integration and planning on future regular sponsorship. Optasy appeared out of the blue with a $125/month sponsorship. 

### Futures:
* As DDEV matures, a clear need will be for more formal governance, likely including a board. 
* DDEV architecture should be a theme in 2024, including discussions of swappable webserver (nginx+apache+caddy+???) and swappable interpreter (php-fpm+gunicorn+arbitrary-node.js+???)
* A GUI wrapper for DDEV has long been a pipe dream, but still would help many people.
* A TUI for DDEV might really help people, or improving the current CLI with [bubbletea](https://github.com/charmbracelet/bubbletea) or something.

* Anything on anybody's mind!

## Meeting Notes

https://docs.google.com/document/d/131_iPGV1xbrtfKfXreyTpbhMFN3EcCJlQaG7b6r-Jds/edit?usp=sharing

## Recording

See [Recording](https://www.dropbox.com/scl/fi/eisgl4zvwg3niu8unnvzf/2023-11-01-DDEV-Advisory-Group.mp4?rlkey=z02yax3trhxxf2piss157gby6&dl=0)

## Financial reports

[DDEV Foundation Balance Sheet 2023-10-30-21_51.pdf](https://github.com/ddev/ddev/files/13210123/DDEV.Foundation.Balance.Sheet.2023-10-30-21_51.pdf)
[DDEV Foundation Profit and Loss 2023-10-30-21_50.pdf](https://github.com/ddev/ddev/files/13210124/DDEV.Foundation.Profit.and.Loss.2023-10-30-21_50.pdf)

**Comment by jonaseberle** [comment]
I am sorry I forgot the date although I had it in the calendar.

**Reply by rfay** [reply]
Next time! Always great to see you. Thanks for all your contributions in all the places.
